test/server/lib/validation/index.test.js

```diff
@@ -65,14 +63,22 @@ describe('server/lib/validation/index', () => {
     expect(validator.isValidUrl(null)).to.be.false;
     expect(stubLogger).to.have.been.calledThrice;
   });
-
-  it.skip('should validate a correctly formatted domain', () => {
+  it('should validate a correctly formatted Redirect Uri', () => {
+    expect(validator.isValidRedirectUri('localhost:3000/user/callback')).to.be.true;
+    expect(validator.isValidRedirectUri('https://example.com')).to.be.true;
+    expect(stubLogger).to.have.been.calledTwice;
+  });
+  it('should validate and return an error for an blank or Redirect Uri', () => {
+    expect(validator.isValidRedirectUri('@@!!()*&^%£$""><???~~`')).to.be.false;
+    expect(validator.isValidRedirectUri('')).to.be.false;
+    expect(stubLogger).to.have.been.calledTwice;
+  });
```

Comment:
Names not right, would 'should validate and return an error for an blank or invalid characters' be nor accurate?

Reply:
Ive pushed up the change :)